WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift

```diff
@@ -117,7 +118,13 @@ class NotificationSettingDetailsViewController: UITableViewController {
     }
 
     @IBAction func reloadTable() {
-        sections = isDeviceStreamDisabled() ? sectionsForDisabledDeviceStream() : sectionsForSettings(settings!, stream: stream!)
+        if isDeviceStreamDisabled() {
+            sections = sectionsForDisabledDeviceStream()
+        } else if isDeviceStreamUnknown() {
+            sections = sectionsForUnkownDeviceStream()
+        } else {
+            sections = sectionsForSettings(settings!, stream: stream!)
```

Comment:
Can we unwrap these `!`s?

Reply:
Sure—I always like the opportunity to remove some `!` from old code! 👍

Reply:
! 💥